### ISSUE TITLE
[BUG] Paper 1.12.2 WG Issue

### ISSUE THREAD

**Issue by FrozenBeard** [eldoriarpg/BigDoorOpener#26]
Breaks on loading on 1.12.2 with WG version.
```
[18:20:54] [Server thread/INFO]: [BigDoorsOpener] Enabling BigDoorsOpener v2.1.1-BETA
[18:20:54] [Server thread/INFO]: [BigDoorsOpener] Hooked into Big Doors successfully.
[18:20:54] [Server thread/INFO]: [BigDoorsOpener] World Guard found. Trying to get a hook.
[18:20:54] [Server thread/ERROR]: Error occurred while enabling BigDoorsOpener v2.1.1-BETA (Is it up to date?)
java.lang.NoClassDefFoundError: com/sk89q/worldguard/WorldGuard
	at de.eldoria.bigdoorsopener.BigDoorsOpener.loadExternalSources(BigDoorsOpener.java:176) ~[?:?]
	at de.eldoria.bigdoorsopener.BigDoorsOpener.onEnable(BigDoorsOpener.java:96) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:395) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:344) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:442) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:403) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:341) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:289) ~[patched_1.12.2.jar:git-Paper-1617]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:616) ~[patched_1.12.2.jar:git-Paper-1617]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: com.sk89q.worldguard.WorldGuard
	at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:156) ~[patched_1.12.2.jar:git-Paper-1617]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:104) ~[patched_1.12.2.jar:git-Paper-1617]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:589) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:522) ~[?:?]
	... 13 more
[18:20:54] [Server thread/INFO]: [BigDoorsOpener] Disabling BigDoorsOpener v2.1.1-BETA
```

**Comment by RainbowDashLabs** [comment]
Which World Guard Version do you use?

**Comment by RainbowDashLabs** [comment]
Ok. I dived into the code of the old World Guard and it would be a loot of work which would end up in two separate versions of the plugin or a illegal reflection chain which would result in a horrible performance.

So I am sad to inform you that there will no World Guard support for version on 1.12 and below.

I will add a simple own implementation of a cuboid region which you can use instead of a world guard region.
I will also keep the workaround I gave you. So world guard integration will just be disabled on a server with a world guard region below 7

**Comment by RainbowDashLabs** [comment]
Closed in #27
